lib/monotime.h

```diff
@@ -59,6 +59,18 @@ struct printfrr_eargs;
 	} while (0)
 #endif
 
+#ifndef ms2ts
+#define ms2ts(ms, ts, tsp)                                            \
```

Comment:
> this is a little opaque - could we have "msec_to_timespec" instead?
sure

> but really: why do we need this at all - why 'pwait2()' instead of 'pwait()' ?
Indeed pwait and pwait2 do same work. Using either is fine.